unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/MyAccountHUD/MyProfileLinkListComponentView.cs

```diff
@@ -77,7 +75,7 @@ public void EnableOrDisableAddNewLinkOption(bool enabled)
         private void EnableOrDisableAddButton()
         {
             addButton.interactable = newLinkTitle.text.Length > 0 && newLinkUrl.text.Length > 0
-                                                                  && httpRegex.IsMatch(newLinkUrl.text)
+                                                                  && LinkValidator.IsValid(newLinkUrl.text)
```

Comment:
I wounder what happens if the profile already has broken links and you try to change your wearables through the backpack (which triggers a profile deployment), since we are only fixing links from "my account" menu. If the profile is rejected, then we should make this validation on a lower level and filter out those links that dont meet the requirements

Reply:
Good point, but they will have the workaround to just delete those links from their profile